installnode.sh

```diff
@@ -48,63 +47,49 @@ FIG
 
 # Check if executed as root user
 if [[ $EUID -ne 0 ]]; then
-	echo -e "This script has to be run as \033[1mroot\033[0m user."
-	exit 1
+  echo -e "This script has to be run as \033[1mroot\033[0m user."
+  exit 1
 fi
 
-
 # Print variable on a screen
 decho "Make sure you double check information before hitting enter!"
 
-read -e -p "Please enter the domain where the Kadena server will run: " whereami
-if [[ "$whereami" == "" ]]; then
-	decho "WARNING: No domain entered, exiting!!!"
-	exit 3
+# --- USER INPUT --- #
+read -e -p "Please enter your node's Domain Name: " whereami
+if [[ $whereami == "" ]]; then
+    decho "WARNING: No domain given, exiting!"
+    exit 3
 fi
 
-# Check for systemd
-systemctl --version >/dev/null 2>&1 || { decho "systemd is required. Are you using Ubuntu 18.04?"  >&2; exit 1; }
-
-
-# Install swap
-decho "Enabling a swap partition..." 
+read -e -p "Please enter your Public Key: " publickey
+if [[ $publickey == "" ]]; then
+    decho "WARNING: No public key given, exiting!"
+    exit 3
+fi
 
-if free | awk '/^Swap:/ {exit !$2}'; then
-	decho "Has swap..."
-else
-	touch /var/swap.img
-	chmod 600 /var/swap.img
-	dd if=/dev/zero of=/var/swap.img bs=1024k count=2048
-	mkswap /var/swap.img
-	swapon /var/swap.img
-	echo "/var/swap.img none swap sw 0 0" >> /etc/fstab
+read -e -p "Please enter your Email Address: " email
+if [[ $email == "" ]]; then
+    decho "WARNING: No email adderss given, exiting!"
+    exit 3
 fi
 
+# --- SYSTEM SETUP --- #
 
-# Update packages
-decho "Updating system..."   
+# Check for systemd.
+systemctl --version >/dev/null 2>&1 || { decho "systemd is required. Are you using Ubuntu 18.04?" >&2; exit 1; }
+
+# Update packages.
+decho "Updating system..."
 
 apt-get update -y >> $LOG_FILE 2>&1
-dpkg --configure -a
 
 # Install required packages
 decho "Installing base packages and dependencies..."
 decho "This may take a while..."
 
 apt-get install -y certbot >> $LOG_FILE 2>&1
-apt-get install -y librocksdb5.8  >> $LOG_FILE 2>&1
-apt-get install -y curl  >> $LOG_FILE 2>&1
-
-decho "Installing ufw..."
-apt-get install -y ufw >> $LOG_FILE 2>&1
-ufw allow ssh/tcp >> $LOG_FILE 2>&1
-ufw allow sftp/tcp >> $LOG_FILE 2>&1
-ufw allow 80/tcp >> $LOG_FILE 2>&1
-ufw allow 443/tcp >> $LOG_FILE 2>&1
-ufw default deny incoming >> $LOG_FILE 2>&1
-ufw default allow outgoing >> $LOG_FILE 2>&1
-ufw logging on >> $LOG_FILE 2>&1
-ufw --force enable >> $LOG_FILE 2>&1
```

Comment:
Leaving this out - it may have been contributing to the connection issues we were seeing.